pkgs/development/python-modules/openai-triton/default.nix

```diff
@@ -111,6 +112,11 @@ buildPythonPackage rec {
     # Use our linker flags
     substituteInPlace python/triton/common/build.py \
       --replace '${oldStr}' '${newStr}'
+    # triton/common/build.py will be called both on build, and sometimes in runtime. This is cursed.
+    substituteInPlace python/triton/common/build.py \
+      --replace 'os.getenv("TRITON_LIBCUDA_PATH")' '"${cudaPackages.cuda_cudart}/lib"'
+    substituteInPlace python/triton/common/build.py \
+      --replace 'os.environ.get("CC")' '"${cudaPackages.backendStdenv.cc}/bin/cc"'
```

Comment:
I would suggest making this (and the above) something like `os.environ.get("CC", default=${...})`. I would also suggest upstream to make this `TRITON_CC` instead of `CC`, to avoid unintentionally changing the compiler